### ISSUE TITLE
predict error, but worked in v0.1.35

### ISSUE THREAD

**Issue by yohacai** [tensorflow/tfjs#964]
To get help from the community, check out our [Google group](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs).

#### TensorFlow.js version
0.14.0
#### Browser version
chrome lastest
#### Describe the problem or feature request
Uncaught (in promise) Error: Nonexistent key: concatenate_71/concatenate_71
#### Code to reproduce the bug / link to feature request
{
            "class_name": "Concatenate",
            "inbound_nodes": [
              [["concatenate_70", 0, 0, {}], ["conv2d_80", 0, 0, {}]]
            ],
            "config": {
              "trainable": true,
              "name": "concatenate_71",
              "axis": -1
            },
            "name": "concatenate_71"
          },
          {
            "class_name": "BatchNormalization",
            "inbound_nodes": [[["concatenate_71", 0, 0, {}]]],
            "config": {
              "gamma_initializer": { "class_name": "Ones", "config": {} },
              "moving_mean_initializer": {
                "class_name": "Zeros",
                "config": {}
              },
              "name": "batch_normalization_80",
              "epsilon": 1.1e-5,
              "trainable": true,
              "center": true,
              "moving_variance_initializer": {
                "class_name": "Ones",
                "config": {}
              },

**Comment by caisq** [comment]
@yohacai Thanks for reporting this issue. The JSON you provided doesn't seem to be the whole model JSON. Can you provide the whole model JSON (weights are not needed)?

**Comment by yohacai** [comment]
It's an OCR model.But I got different result between tfjs(v0.13.5) and keras.

**Comment by caisq** [comment]
@yohacai Thanks again for providing the JSON. Sorry for the breakage introduced in 0.14.0. We've opened a pull request to fix the underlying problem (see the PR linked above).

As for different results, I tried running `predict()` on your model with randomly initialized weights, in both Python Keras and TensorFlow.js (Node). I got mostly consistent results (except some small differences that may have to do with numerical precision issues). See examples below:

Python:
     [[[0.00019427 0.00018104 0.00018765 ... 0.00019013 0.00013914 0.0001726 ]]]
JS:
     [ [[0.0001943, 0.000181, 0.0001877, ..., 0.0001901, 0.0001391, 0.0001726],]]

Python:	
     [[[0.00021124 0.00012701 0.00012853 ... 0.00016232 0.0001846  0.00021724]]]
JS:
     [ [[0.0002112, 0.000127, 0.0001285, ..., 0.0001623, 0.0001846, 0.0002172],]]

I wonder whether the differences you are seeing between Python and JS is bigger than this? If so, we can look into it further.

**Comment by yohacai** [comment]
![image](https://user-images.githubusercontent.com/3947925/49628778-2ec00100-fa21-11e8-9215-6478aa76051d.png)

**Comment by yohacai** [comment]
tfjs-core@0.14.1

**Comment by caisq** [comment]
@yohacai The fix to the execution ("Nonexistent key") error is still under review in the PR above and hasn't been published yet. The error you showed seems to be unrelated to that issue and may have something to do with your custom linking. Can you wait for the next release of @tensorflow/tfjs, which will have version number 0.14.1? Thanks.

**Comment by caisq** [comment]
@yohacai The issue should have been fixed in the just released 0.14.1 (https://www.npmjs.com/package/@tensorflow/tfjs). I tested loading your model (with random weights) and running predict() on it in the browser. The result I got for an all-one input seems to match the result from Python Keras.

Please give it a try and let me know if it works.

**Comment by caisq** [comment]
@yohacai Since the result mismatch issue isn't directly related to this issue anymore, can you do the following if you want to work with us to debug the mismatch?
1. Open a new issue
2. Provide a .h5 file that contains both the model config and weights for us to convert. Or equivalently, provide the model.json and weight files in the TF.js format.
3. Describe the input you use and the mismatch in the outputs from Python and JavaScript. Thanks.
